src/main/java/com/microsoft/aad/msal4j/AuthorizationCodeParameters.java

```diff
@@ -22,16 +22,27 @@
 @AllArgsConstructor(access = AccessLevel.PRIVATE)
 public class AuthorizationCodeParameters {
 
+    /**
+     * Authorization code acquired in the first step of OAuth2.0 authorization code flow. For more
+     * details, see https://aka.ms/msal4j-authorization-code-flow
```

Comment:
Can we update this aka.ms link to 
https://docs.microsoft.com/en-us/azure/active-directory/develop/scenario-desktop-acquire-token?tabs=java#acquire-a-token-interactively